cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -330,13 +330,13 @@ impl ASTNode<Option<cst::Ident>> {
             | cst::Ident::In
             | cst::Ident::Has
             | cst::Ident::Like => {
-                errs.push(err::ParseError::ToAST(format!(
+                errs.push(err::ParseError::InvalidIdentifier(format!(
                     "This identifier is reserved and cannot be used: {ident}"
```

Comment:
Maybe reorder this one to something like "{ident}. This identifier is reserved and cannot be used" so that it concatenates nicely with "invalid identifier:"